data-platform/jupyter-notebook/FoxgloveDataPlatform.ipynb

```diff
@@ -306,55 +306,20 @@
    "outputs": [],
    "source": [
     "import matplotlib as mpl\n",
-    "from pandasql import sqldf\n",
-    "\n",
-    "color_to_classname = {\n",
-    "    \"#000000\": \"noise\",\n",
-    "    \"#468250\": \"animal\",\n",
-    "    \"#0000e6\": \"human.pedestrian.adult\",\n",
-    "    \"#87ceeb\": \"human.pedestrian.child\",\n",
-    "    \"#f08080\": \"human.pedestrian.construction_worker\",\n",
-    "    \"#db7093\": \"human.pedestrian.personal_mobility\",\n",
-    "    \"#000080\": \"human.pedestrian.police_officer\",\n",
-    "    \"#f08080\": \"human.pedestrian.stroller\",\n",
-    "    \"#8a2be2\": \"human.pedestrian.wheelchair\",\n",
-    "    \"#708090\": \"movable_object.barrier\",\n",
-    "    \"#d2691e\": \"movable_object.debris\",\n",
-    "    \"#696969\": \"movable_object.pushable_pullable\",\n",
-    "    \"#2f4f4f\": \"movable_object.trafficcone\",\n",
-    "    \"#bc8f8f\": \"static_object.bicycle_rack\",\n",
-    "    \"#dc143c\": \"vehicle.bicycle\",\n",
-    "    \"#ff7f50\": \"vehicle.bus.bendy\",\n",
-    "    \"#ff4500\": \"vehicle.bus.rigid\",\n",
-    "    \"#ff9e00\": \"vehicle.car\",\n",
-    "    \"#e99646\": \"vehicle.construction\",\n",
-    "    \"#ffd700\": \"vehicle.emergency.ambulance\",\n",
-    "    \"#ffd700\": \"vehicle.emergency.police\",\n",
-    "    \"#ff3d63\": \"vehicle.motorcycle\",\n",
-    "    \"#ff8c00\": \"vehicle.trailer\",\n",
-    "    \"#ff6347\": \"vehicle.truck\",\n",
-    "    \"#00cfbf\": \"flat.driveable_surface\",\n",
-    "    \"#af004b\": \"flat.other\",\n",
-    "    \"#4b004b\": \"flat.sidewalk\",\n",
-    "    \"#70b43c\": \"flat.terrain\",\n",
-    "    \"#deb887\": \"static.manmade\",\n",
-    "    \"#ffe4c4\": \"static.other\",\n",
-    "    \"#00af00\": \"static.vegetation\",\n",
-    "    \"#fff0f5\": \"vehicle.ego\",\n",
-    "}\n",
+    "import duckdb\n",
     "\n",
     "flattened_markers = []\n",
     "for topic, record, message in marker_messages:\n",
-    "    for marker in message.markers:\n",
-    "        color = mpl.colors.to_hex([marker.color.r, marker.color.g, marker.color.b])\n",
-    "        class_name = color_to_classname[color]\n",
-    "        flattened_markers.append((marker.text, class_name))\n",
+    "    for entity in message.entities:\n",
+    "        for kv in entity.metadata:\n",
+    "            if kv.key == \"category\":\n",
+    "                flattened_markers.append((entity.id, kv.value))\n",
+    "\n",
     "annotations = pd.DataFrame(flattened_markers, columns=[\"annotation_id\", \"class_name\"])\n",
     "\n",
     "pysqldf = lambda q: sqldf(q, globals())\n",
```

Comment:
looks like this this can be deleted — @wimagguc? (https://github.com/foxglove/jupyter-data-platform/pull/18)

Reply:
Yeah let's not keep two repos for the same thing :)

Reply:
just in case... I meant delete the line above which assigns `pysqldf`, since it's no longer used as of that linked PR.

Reply:
Oh interesting, you're right. Seems like `pysqldf` is not used any longer. (Sorry, all the highlights and links threw me off, I looked at the wrong line earlier..)